policy/pa.go

```diff
@@ -174,24 +174,24 @@ func isDNSCharacter(ch byte) bool {
-	errEmptyName            = berrors.MalformedError("DNS name was empty")
-	errNameEndsInDot        = berrors.MalformedError("DNS name ends in a period")
-	errTooFewLabels         = berrors.MalformedError("DNS name does not have enough labels")
-	errLabelTooShort        = berrors.MalformedError("DNS label is too short")
-	errLabelTooLong         = berrors.MalformedError("DNS label is too long")
-	errMalformedIDN         = berrors.MalformedError("DNS label contains malformed punycode")
-	errInvalidRLDH          = berrors.RejectedIdentifierError("DNS name contains a R-LDH label")
-	errTooManyWildcards     = berrors.MalformedError("DNS name had more than one wildcard")
-	errMalformedWildcard    = berrors.MalformedError("DNS name had a malformed wildcard label")
-	errICANNTLDWildcard     = berrors.MalformedError("DNS name was a wildcard for an ICANN TLD")
-	errWildcardNotSupported = berrors.MalformedError("Wildcard names not supported")
+	errNonPublic            = berrors.MalformedError("Domain name does not end with a valid public suffix (TLD)")
+	errICANNTLD             = berrors.MalformedError("Domain name ends with a reserved ICANN TLD. Certificate can not be issued")
+	errPolicyForbidden      = berrors.RejectedIdentifierError("Policy forbids issuing a certificate for the domain name. Part or all of the name may be blacklisted")
+	errInvalidDNSCharacter  = berrors.MalformedError("Domain name contains invalid character")
+	errNameTooLong          = berrors.MalformedError("Domain name too long. (Note that certificates containing domain names over 230 octets long are not currently supported)")
+	errIPAddress            = berrors.MalformedError("Certificate issue for IP addresses is not supported")
+	errTooManyLabels        = berrors.MalformedError("Domain name has too many labels. Domain names can have up to 10 labels (parts)")
+	errEmptyName            = berrors.MalformedError("Domain name is empty")
+	errNameEndsInDot        = berrors.MalformedError("Domain name ends in a period")
+	errTooFewLabels         = berrors.MalformedError("Domain name does not have enough labels (parts). At least two parts are required for a valid domain name")
+	errLabelTooShort        = berrors.MalformedError("Domain label is too short. Each part of the domain name must be between 1 and 63 octets (bytes) long")
+	errLabelTooLong         = berrors.MalformedError("Domain label too long. Each part of the domain name must be between 1 and 63 octets (bytes) long")
+	errMalformedIDN         = berrors.MalformedError("Domain name contains malformed punycode")
+	errInvalidRLDH          = berrors.RejectedIdentifierError("Domain name contains a R-LDH label")
+	errTooManyWildcards     = berrors.MalformedError("Domain name has more than one wildcard")
+	errMalformedWildcard    = berrors.MalformedError("Domain name has a malformed wildcard label")
+	errICANNTLDWildcard     = berrors.MalformedError("Domain name is a wildcard for a reserved ICANN TLD. Certificate can not be issued")
```

Comment:
```suggestion
	errICANNTLDWildcard     = berrors.MalformedError("Domain name is a wildcard for an ICANN TLD")
```

Reply:
Edited, and made consistent with other similar err messages. Take a look now